Test/RadixSort/main.cpp

```diff
@@ -52,9 +52,9 @@ using u32 = Oro::RadixSort::u32;
 class SortTest
 {
   public:
-	SortTest( oroDevice dev, oroCtx ctx ) : m_device( dev ), m_ctx( ctx )
+	SortTest( oroDevice dev, oroCtx ctx, OrochiUtils* oroutils ) : m_device( dev ), m_ctx( ctx )
```

Comment:
(same reason as above) Use reference instead of pointer since this cannot be null and yet we do not check for nullptr in the code.

```suggestion
	SortTest( oroDevice dev, oroCtx ctx, OrochiUtils& oroutils ) : m_device( dev ), m_ctx( ctx )
```